example_files/example_project.json

```diff
@@ -118,7 +118,11 @@
         "building_type": "Food service",
         "floor_area": 22313,
         "footprint_area": 22313,
-        "number_of_stories": 1
+        "number_of_stories": 1,
+        "ev_charging": true,
```

Comment:
`ev_charging` needs to be set to true when a user wants to add ev_charging to the feature. 
When set to true, users can provide additional arguments for the ev_charging measure or rely on the defaults used in the measure.